tests/benchmark/array_access_i32.ts

```diff
@@ -3,6 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
  */
 
+type i32 = number;
```

Comment:
Do we require app to write this line?

Reply:
For compiling ts to wasm, we don't need this line, but we also need to compile ts to js, in this situation, we need to set i32 as a resonable type.